crates/transaction-pool/src/metrics.rs

```diff
@@ -38,6 +38,16 @@ pub struct TxPoolMetrics {
     pub(crate) performed_state_updates: Counter,
 }
 
+/// Transaction pool blobstore metrics
+#[derive(Metrics)]
+#[metrics(scope = "transaction_pool")]
+pub struct BlobStoreMetrics {
+    /// Number of failed inserts into the blobstore
```

Comment:
Should we also have other metrics? Blob size, inserts, removes etc. (not only failures)

Reply:
will definitely add

but would like to include these prs asap